### PR TITLE
i18n(es): add `no-prerendered-routes-with-domains`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Translated `no-prerendered-routes-with-domains` to spanish 

#### Related issues & labels (optional)

- Closes # N/A
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/reference/errors/no-prerendered-routes-with-domains.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Prerendered routes aren't supported when internationalization domains are enabled.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+
+> **NoPrerenderedRoutesWithDomains**: Las páginas estáticas aún no son compatibles con múltiples dominios. Si desea habilitar esta función, debe deshabilitar el prerenderizado para la página COMPONENT
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/no-prerendered-routes-with-domains.mdx:8]:
```suggestion
> **NoPrerenderedRoutesWithDomains**: Las páginas estáticas aún no son compatibles con múltiples dominios. Si deseas habilitar esta función, debes deshabilitar el prerenderizado para la página COMPONENT
```

File: src/content/docs/es/reference/errors/no-prerendered-routes-with-domains.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## ¿Qué salió mal?
+Las páginas estáticas aún no son compatibles con múltiples dominios. Si desea habilitar esta función, debe deshabilitar el prerenderizado.
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/no-prerendered-routes-with-domains.mdx:11]:
```suggestion
Las páginas estáticas aún no son compatibles con múltiples dominios. Si deseas habilitar esta función, debes deshabilitar el prerenderizado.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Pull request merged**